src/subscriber/Subscriber.ts

```diff
@@ -89,8 +91,14 @@ export class Subscriber extends OTEventEmitter<{
       .on("participant-left", (event?: DailyEventObjectParticipantLeft) => {
         if (!event?.participant) return;
         this.eventHandler.onParticipantLeft(event.participant.session_id);
+      })
+      .once("participant-updated", (dailyEvent) => {
+        if (!dailyEvent) return;
+        // Create stream and add it to subscriber
+        const stream = createStream(dailyEvent.participant);
```

Comment:
A subscriber may be updated for any number of reasons that don't have to do with streams or tracks (eg a name change, or a user data change). Do we definitely want to create a new stream for every kind of update here?